ui/apos/apps/index.js

```diff
@@ -1,12 +1,16 @@
 export default () => {
-  window.apos.util.onReady(openExportUrl);
+  let ready = false;
 
-  function openExportUrl() {
-    window.apos.bus && window.apos.bus.$on('export-download', event => {
-      if (!event.url) {
-        return;
-      }
+  window.apos.util.onReady(() => {
```

Comment:
apos being a global variable you could remove window (or wait for my PR I added it to esling config)